source/table_view.tsx

```diff
@@ -65,6 +100,46 @@ export class TableView extends React.Component<Properties> {
             className={this.props.className}>
           {tableRows}
         </tbody>
-      </table>);
+      </table>
+      </div>);
+  }
+
+  public get columnCount() {
+    return this.props.model.columnCount;
+  }
+
+  public get activeWidth() {
+    return this.props.activeWidth;
   }
+
+  public getColumnRect(index: number): Rectangle {
+    return ({  
+      x: this._header_refs[index].getBoundingClientRect().left,
+      y: this._header_refs[index].getBoundingClientRect().top,
+      width: this._header_refs[index].getBoundingClientRect().width,
+      height: this._header_refs[index].getBoundingClientRect().height,
+      top: this._header_refs[index].getBoundingClientRect().top,
+      left: this._header_refs[index].getBoundingClientRect().left,
+      bottom: this._header_refs[index].getBoundingClientRect().bottom,
+      right: this._header_refs[index].getBoundingClientRect().right
+    } as Rectangle);
+  }
+
+  public onResize(columnIndex: number, width: number) {
+    this._header_refs[columnIndex].style.width = `${width}px`;
+    this._header_refs[columnIndex].style.minWidth = `${width}px`;
+    this._header_refs[columnIndex].style.maxWidth = `${width}px`;
+  }
+
+  public showCursor() {
+    this._header_row_ref.style.cursor = 'col-resize';
+  }
+
+  public hideCursor() {
+    this._header_row_ref.style.cursor = 'auto';
+  } 
+
+  private _header_refs: HTMLHeadElement[];
```

Comment:
Only prefix a field name with an underscore if there is a public property sharing the same name.